### PR TITLE
Features/store the rate of team member

### PR DESCRIPTION
**Pull request by gulshan81**
Targets #2163

### Description
store the rate column data in database.

### Checklist:
- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/Project/Resources/views/subviews/edit-project-team-members.blade.php [hunk 1]
```diff
@@ -70,6 +73,9 @@
                     <div class="col-1 monthly-effort-div">
                         <input type="number" @input="updatedDailyExpectedEffort($event, index, workingDaysInMonth)" :value="projectTeamMember.pivot.daily_expected_effort*workingDaysInMonth" class="form-control monthly-effort">
                     </div>
+                    <div class="col-1">
+                        <input type="number" step="0.01" :name="`project_team_member[${index}][rate]`" v-model="projectTeamMember.pivot.rate" class="form-control">
```
Comment (srivastava-jyoti) [inline Modules/Project/Resources/views/subviews/edit-project-team-members.blade.php:77]: Why are considering as `step="0.01"`?

Comment (gulshan81) [reply Modules/Project/Resources/views/subviews/edit-project-team-members.blade.php:78]: step="0.01"  will allow only 2 values after the decimal.
ex-0.22

**Review by srivastava-jyoti — APPROVED** [review]

**Comment by gautam071** [comment]
@gulshan81 The daily effort input field is appearing twice. Can you take a look at it once
Screenshot 2022-08-16 at 7 48 09 PM

File: Modules/Project/Resources/views/edit.blade.php [hunk 2]
```diff
@@ -155,6 +155,7 @@ class="text-theme-body text-decoration-none mb-2 mb-xl-4 d-inline-flex align-ite
 
                 updatedDailyExpectedEffort($event, index, numberOfDays) {
                     value = $event.target.value;
+                    this.projectTeamMembers[index]['pivot']['daily_expected_effort'] = value/numberOfDays;
```
Comment (gautam071) [inline Modules/Project/Resources/views/edit.blade.php:158]: Why is this calculation needed?

I don't see anything in the issue, Am I missing something?

**Review by gautam071 — COMMENTED** [review]
Looks good.
One question ?

**Comment by gautam071** [comment]
Also resolve the conflicts

**Pull request closed without merging**